app/controllers/media_controller.rb

```diff
@@ -0,0 +1,32 @@
+class MediaController < ApplicationController
+  def index
+@books = Book.all
+@movies = Movie.all
+@albums = Album.all
```

Comment:
These three lines (3-5) should be indented by two levels (to be one level further indented from line 2).